addons/compat_gm/config.cpp

```diff
@@ -6,15 +6,15 @@ class CfgPatches {
         units[] = {};
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
-        requiredAddons[] = {"ace_csw", "ace_explosives", "ace_refuel", "gm_core"};
+        requiredAddons[] = {"gm_core"};
```

Comment:
I really don't know if we need this, but I added it for some of the other compats
```suggestion
        requiredAddons[] = {"ace_common", "gm_core"};
```

Reply:
Good to have I suppose